dev/TeachingTip/TestUI/TeachingTipPage.xaml.cs

```diff
@@ -35,8 +36,17 @@ enum TipLocation
         VisualTree = 0,
         Resources = 1
     }
-    public sealed partial class TeachingTipPage : TestPage
+    public sealed partial class TeachingTipPage : TestPage, INotifyPropertyChanged
     {
+        public event PropertyChangedEventHandler PropertyChanged;
+        private void NotifyPropertyChanged(string propertyName)
+        {
+            if (PropertyChanged != null)
+            {
+                PropertyChanged(this, new PropertyChangedEventArgs(propertyName));
+            }
```

Comment:
PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName))

Is what VS suggests to me these days for code like this - more concise.

Reply:
Changed.